### ISSUE TITLE
Certificate problem with wp core download 

### ISSUE THREAD

**Issue by jubstuff** [wp-cli/wp-cli#566]
Hello,
I'm trying to use wp-cli on Linux Mint 15. I've managed to install it, but when I issue the command `wp core download` this is what I get:

```
Downloading latest WordPress (en_US)...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: self signed certificate in certificate chain
More details here: http://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now

```

Is there something I could do?

Thank you for this fantastic tool :)

**Comment by SiteDesignUSA** [comment]
I'm sorry about bringing this up but I've googled my issue and this is what comes up. I thought about starting a new bug <rant>but, man, does it really need to be that complicated? </rant>

## SSL certificate problem: self signed certificate in certificate chain

### Bottom Line
Are all the certs in proper order? Why am I getting this error today but didn't 3 days ago. Wp-cli works on my remote servers but stopped (wp core at least) on my local dev machine today.

If the certs are all good to go, OK. Just wondering what's going on.

Here is what core download shows (over and over) today. This worked 3 days ago on the same machine and same network...

<pre>
$ php wp-cli.phar core download --path=/e/dev/rubydoodlewebsites/testing4/wp/ --version=5.2.2 --skip-content
Downloading WordPress 5.2.2 (en_US)...
Error: Failed to get url 'https://wordpress.org/wordpress-5.2.2.zip': cURL error 18: transfer closed with 8910426 bytes remaining to read.
</pre>

So I tried curl directly and am getting this:

<pre>
$ curl -o wp https://wordpress.org/wordpress-5.2.2.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: self signed certificate in certificate chain
More details here: https://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
</pre>

I've tried with 5.1.1 also, but same thing. Wp-cli is using the cache of 4.9.10, here's what that looks like;
<pre>
$ php wp-cli.phar core download --path=/e/dev/rubydoodlewebsites/testing4/wp/ --version=4.9.10 --skip-content
Downloading WordPress 4.9.10 (en_US)...
Using cached file 'C:\Users\tj/.wp-cli/cache/core/wordpress-4.9.10-en_US.zip'...
Success: WordPress downloaded.
</pre>

 so I'll use that for now, but would like to upgrade to 5.

### Background

Using on Windows 10 in git bash, also tried phpstorm's terminal... hoorah!

**Comment by abdullah1908** [comment]
I got the same error, try the command again `wp core download` & it worked for me 😀
